### DISCUSSION TITLE
Intel Arc update broke SDnext

### DISCUSSION THREAD

**Discussion by ethen-sanchez** [vladmandic/sdnext#2306] [Q&A] [answered]
I updated Intel Arc graphics and since then, I can't generate a single image. I get an error that "Connection Errored Out".
running in debug doesn't display any extra messages either. It just stops with no kind of unusual message. I completely removed automatic from my computer and gone through the whole setup of creating a conda environment and setting everything up again but it does the exact same thing. Does anyone have any ideas of something I may have missed?

**Comment by Mindset-Official** [comment, marked as answer]
In windows? The latest drivers broke Ipex, you need to be on 4676 or earlier.  I have reported the issue with intel and the ipex team.

**Reply by ethen-sanchez** [reply]
Thank you for the info. I figured it was the update that messed it up. I reverted to 4676 and it resolved.
